inc/commonglpi.class.php

```diff
@@ -690,6 +690,10 @@ function showTabsContent($options = []) {
          if ($this->getType() == 'Ticket') {
             $this->input = $cleaned_options;
             $this->saveInput();
+            if (isset($cleaned_options['is_locked'])) {
```

Comment:
option is currently called 'locked' and not 'is_locked', see https://github.com/glpi-project/glpi/blob/8b53585c0930deb5bd8519a23c81af3746ed65b1/inc/objectlock.class.php#L454